src/Search/Console.php

```diff
@@ -119,6 +122,75 @@ private function enqueue($type, $id)
         $this->logger->info('Item added successfully.');
     }
 
+    public function indexRebuildCommand(){
+
+        $newIndexName = 'elife_search_tmp';
+        $client = $this->getElasticClient();
+
+        // Delete the existing 'elife_search_tmp' index if we have one
+        echo "checking if an old 'elife_search_tmp' index exists \n";
+        $response = $client->deleteIndex($newIndexName);
+
+        if ($response['payload'] instanceof \eLife\Search\Api\Elasticsearch\Response\SuccessResponse){
+
+            echo "Found an old 'elife_search_tmp' index, Deleted it \n";
+
+        } else if ($response['payload'] instanceof \eLife\Search\Api\Elasticsearch\Response\ErrorResponse){
+
+            if ($response['payload']->error['type'] == 'index_not_found_exception'){
+
+                echo "Did not find an old 'elife_search_tmp' index, This is okay .. continuing \n";
+
+            }else{
+
+                echo "Something went wrong, we got an exception we were not expecting \n";
+
+            }
+        }
+
+        echo "Creating a new (empty) 'elife_search_tmp' index  \n";
+        $response = $client->createIndex($newIndexName);
+
+        if ($response['payload'] instanceof \eLife\Search\Api\Elasticsearch\Response\SuccessResponse){
+
+            echo "Successfully created the new index \n";
+
+        }else {
+
+            echo "Could not create the new index ... exiting \n";
+            die();
```

Comment:
Instead of dying here, you should be able to `return 1;` where `1` is the exit code